delivery/subscription.mdx

```diff
@@ -83,33 +83,39 @@ If you specify `FULL` instead of the `since_clause`, the subscription cursor sta
 FETCH from cursor function is currently only supported in the PSQL simple query mode. If you are using components like JDBC that default to the extended query mode, please manually set the mode to simple query mode.
 
 </Info>
-#### FETCH NEXT FROM cursor
+#### Non-blocking data fetch
 
-After creating a subscription cursor, you can fetch the data by the `FETCH NEXT FROM cursor_name` command. Then you will see a result like below:
+```sql
+FETCH NEXT/n FROM cursor_name;
+```
+
+Fetch the next row or up to N rows from the cursor. If fewer than N rows are available, it will return whatever is available immediately without waiting. This also means that if there are no rows available (i.e., the latest data has been reached), an empty result will be returned immediately.
 
 ```sql
 FETCH NEXT FROM cur;
 
 ----RESULT
-t1.v1 | t1.v2 | t1.v3 |    t1.op     |  rw_timestamp
+t1.v1 | t1.v2 | t1.v3 |    t1.op     |  rw_timestamp  
 -------+-------+-------+--------------+---------------
-     1 |     1 |     1 | UpdateDelete | 1715669376304
+    1 |     1 |     1 | UpdateDelete | 1715669376304
 (1 row)
 ```
 
-The `op` column in the result stands for the change operations. It has four options: `insert`, `update_insert`, `delete`, and `update_delete`. For a single UPDATE statement, the subscription log will contain two separate rows: one with `update_insert` and another with `update_delete`. This is because RisingWave treats an UPDATE as a delete of the old value followed by an insert of the new value. As for `rw_timestamp`, it corresponds to the Unix timestamp in milliseconds when the data was written.
+In the example above, the `op` column in the result indicates the type of change operations. There are four options: `insert`, `update_insert`, `delete`,  and `update_delete`. For a single UPDATE statement, the subscription log will contain two separate rows: one with `update_insert` and another with `update_delete`. This is because RisingWave treats an UPDATE as a delete of the old value followed by an insert of the new value. As for `rw_timestamp`, it corresponds to the Unix timestamp in milliseconds when the data was written.
```

Comment:
The four [options](https://github.com/risingwavelabs/risingwave/blob/fdeeb2e6c358fe8f04d1fa29ef5384e75ae2e3f8/src/common/src/array/stream_chunk.rs#L95) are in camelCase instead of snake_case: `Insert`, `Delete`, `UpdateInsert`, `UpdateDelete`.